integrations/jira/.port/spec.yaml

```diff
@@ -8,6 +8,8 @@ features:
     resources:
       - kind: project
       - kind: issue
+      - kind: board
```

Comment:
This doesn't affect anything but a bit more about structure. Can you order it in this format
board
sprint
project
issue

Reply:
The order should be in order of how top level they are which is:
- Project
- Board
- Sprint
- Issue

I'll push a commit to fix this